codegen/sdk-codegen/build.gradle.kts

```diff
@@ -117,7 +117,8 @@ fun generateSmithyBuild(services: List<AwsService>): String {
                       "build": {
                           "rootProject": $buildStandaloneSdk
                       },
-                      "useInterceptors": ${interceptorsServices.contains(service.packageName)}
+                      "useInterceptors": ${interceptorsServices.contains(service.packageName)},
+                      "mergeModels": true
```

Comment:
When generating an AWS SDK, models are merged.  This can be changed to `false` for troubleshooting, if desired.